service/datasets/d1.yaml

```diff
@@ -5,6 +5,10 @@ Database:
       UserId:
```

Comment:
All of these YAML files were changed to new syntax in PR #94 You could merge in this change to your PR before submitting.  Or preferable would be to use object literals for these metadata, since object literal interface didn't change, and the code would continue working.

Because we still have more changes to make to the metadata format, it will be best to keep metadata in YAML files or in object literals.

Reply:
`from burdock.query.sql.metadata.metadata import *
import copy

table1 = Table("dbo", "d1", 5000, \
    [\
        String("DeviceID", 0, True),\
        Boolean("Refurbished"), \
        Float("Temperature", 20.0, 70.0)
    ])

table2 = copy.copy(table1)
table2.name = "d2"

x = Database([table1],"csv")
y = Database([table2],"csv")

print(x)
print(y)`

Reply:
Removed the dependence on YAML files and used metadata object literals.